src/components/ui/badge.jsx

```
@@ -0,0 +1,44 @@
+import * as React from "react"
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)